strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/users/UserControllerTestIT.java

```diff
@@ -87,10 +90,49 @@ public void init()
         setContextBaseUrl(getContextBaseUrl() + "/api/users");
     }
 
-    @Test
-    public void testGetUser()
+    @ParameterizedTest
+    @MethodSource("usersProvider")
+    public void testGetUser(String acceptHeader)
     {
-        final String username = "developer01";
+        String username = "test-user-1";
+
+        deleteCreatedUser(username);
+
+        UserDto user = new UserDto();
+        user.setEnabled(true);
+        user.setUsername(username);
+        user.setPassword("test-password");
+        user.setSecurityTokenKey("before");
+
+        UserAccessModelDto userAccessModelDto = new UserAccessModelDto();
```

Comment:
**Cosmetic:** Same as above -- could you please split this into easier to read code blocks like this:

```
PrivilegeDto repositoryPrivilege = new PrivilegeDto("ARTIFACTS_RESOLVE","ARTIFACTS_RESOLVE");

UserAccessModelDto userAccessModelDto = new UserAccessModelDto();UserStorageDto userStorageDto = new UserStorageDto();

UserRepositoryDto userRepositoryDto = new UserRepositoryDto();
userRepositoryDto.setRepositoryId("releases");
userRepositoryDto.getRepositoryPrivileges().add(repositoryPrivilege);

userStorageDto.setStorageId("storage0");
userStorageDto.getRepositories().add(userRepositoryDto);

userAccessModelDto.getStorages().add(userStorageDto);

user.setUserAccessModel(userAccessModelDto);
```

This way, the logic is grouped in easier to follow and read blocks of code.

Reply:
done!